tests/models/test_model.py

```diff
@@ -831,6 +832,43 @@ def test_forward_with_padding(attention_impl: str, pos_emb_config: dict,
                 batched_output[1, :],
                 atol=1e-6 if attention_impl == 'torch' else 1e-8)
 
+        try:
```

Comment:
having try except block in test seems counter-intuitive 🤔 

you probably cant import it on CPU tests. maybe gate by "if not running on gpu"?